modules/ocf_mirrors/manifests/qt.pp

```diff
@@ -0,0 +1,19 @@
+class ocf_mirrors::qt {
+  file {
+    '/opt/mirrors/project/qt':
+      ensure  => directory,
+      source  => 'puppet:///modules/ocf_mirrors/project/qt/',
+      owner   => mirrors,
+      group   => mirrors,
+      mode    => '0755',
+      recurse => true;
+  }
+
+  cron {
+    'qt':
+      command => '/opt/mirrors/project/qt/sync-archive > /dev/null',
+      user    => 'mirrors',
+      hour    => '*/4',
+      minute  => '5';
```

Comment:
I think the idea here is to choose minutes that don't overlap to try and spread out the impact of the mirrors so they don't try to all sync at the same time for instance. We've spread mostly all through the hour, but there's still some unused times especially between 55 and 15, so I think leaving this at minute 5 is good:

```
37 */8 * * * /opt/mirrors/project/apache/sync-archive > /dev/null
22 */3 * * * /opt/mirrors/project/archlinux/sync-archive > /dev/null
22 */3 * * * /opt/mirrors/project/centos/sync-archive > /dev/null
44 */3 * * * /opt/mirrors/project/centos/sync-altarch > /dev/null
41 * * * * /opt/mirrors/project/finnix/sync-releases > /dev/null
38 */4 * * * /opt/mirrors/project/gnu/sync-archive > /dev/null
35 */2 * * * /opt/mirrors/project/manjaro/sync-archive > /dev/null
15 */3 * * * /opt/mirrors/project/parrot/sync > /dev/null
15 * * * * /opt/mirrors/project/tails/sync > /dev/null
53 */2 * * * /opt/mirrors/project/tanglu/sync-releases > /dev/null
18 */7 * * * /opt/mirrors/project/ubuntu/sync-releases > /dev/null
10 * * * * BASEDIR=/opt/mirrors/project/debian /opt/mirrors/project/debian/bin/ftpsync > /dev/null 2>&1
20 * * * * BASEDIR=/opt/mirrors/project/debian-security /opt/mirrors/project/debian-security/bin/ftpsync > /dev/null 2>&1
30 * * * * BASEDIR=/opt/mirrors/project/debian-cd /opt/mirrors/project/debian-cd/bin/ftpsync > /dev/null 2>&1
15 */3 * * * BASEDIR=/opt/mirrors/project/kali /opt/mirrors/project/kali/bin/ftpsync > /dev/null 2>&1
25 * * * * BASEDIR=/opt/mirrors/project/kali-images /opt/mirrors/project/kali-images/bin/ftpsync > /dev/null 2>&1
55 * * * * BASEDIR=/opt/mirrors/project/puppetlabs /opt/mirrors/project/puppetlabs/bin/ftpsync > /dev/null 2>&1
45 * * * * BASEDIR=/opt/mirrors/project/raspbian /opt/mirrors/project/raspbian/bin/ftpsync > /dev/null 2>&1
40 * * * * BASEDIR=/opt/mirrors/project/tanglu /opt/mirrors/project/tanglu/bin/ftpsync > /dev/null 2>&1
45 * * * * BASEDIR=/opt/mirrors/project/trisquel /opt/mirrors/project/trisquel/bin/ftpsync > /dev/null 2>&1
55 * * * * BASEDIR=/opt/mirrors/project/trisquel-images /opt/mirrors/project/trisquel-images/bin/ftpsync > /dev/null 2>&1
50 * * * * BASEDIR=/opt/mirrors/project/ubuntu /opt/mirrors/project/ubuntu/bin/ftpsync > /dev/null 2>&1
```

It probably doesn't matter much as long as it isn't a huge sync every time.